guides/models/creating-updating-and-deleting-records.md

```diff
@@ -88,48 +76,41 @@ person.isAdmin; // => false
 person.changedAttributes(); // => {}
 ```
 
-## Handling Validation Errors
+## Gérer les erreurs de validation
 
-If the backend server returns validation errors after trying to save, they will
-be available on the `errors` property of your model. Here's how you might display
-the errors from saving a blog post in your template:
+Si le serveur _backend_ retourne des erreurs de validations après une tentative de sauvegarde, elles seront disponibles dans la propriété `errors` de votre modèle. Voici un exemple montrant comment vous pouvez afficher les erreurs apparaissant en essayant de sauvegarder un article dans votre _template_&nbsp;:
 
 ```handlebars
 {{#each this.post.errors.title as |error|}}
-  <div class="error">{{error.message}}</div>
+  <div class='error'>{{error.message}}</div>
 {{/each}}
 {{#each this.post.errors.body as |error|}}
-  <div class="error">{{error.message}}</div>
+  <div class='error'>{{error.message}}</div>
 {{/each}}
 ```
 
-## Promises
+## Promesses
 
-[`save()`](https://api.emberjs.com/ember-data/release/classes/Model/methods/save?anchor=save) returns
-a promise, which makes it easy to asynchronously handle success and failure
-scenarios. Here's a common pattern:
+[`save()`](https://api.emberjs.com/ember-data/release/classes/Model/methods/save?anchor=save) retourne une promesse, ce qui facilite la gestion asynchrone des succès et des échecs. Voici un exemple de cas fréquent&nbsp;:
 
 ```javascript
-// Assumed to have already injected the router and store services
+// On suppose que l'on a déjà injecté le router et les services du _store_
```

Comment:
I don't think we should use italic in code comments, the text won't be displayed in italic.
@BlueCutOfficial 

```suggestion
// On suppose que l'on a déjà injecté les services router et store
```

Reply:
Yes, good to me 👍